airflow-core/src/airflow/plugins_manager.py

```diff
@@ -525,7 +525,7 @@ def integrate_macros_plugins() -> None:
         if plugin.name is None:
             raise AirflowPluginException("Invalid plugin name")
 
-        macros_module = make_module(f"airflow.macros.{plugin.name}", plugin.macros)
+        macros_module = make_module(f"airflow.sdk.definitions.{plugin.name}", plugin.macros)
```

Comment:
We should update docs to say, it will now not registered under `airflow.macros.` in [this link](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/plugins.html#example:~:text=%23%20Will%20show%20up%20under%20airflow.macros.test_plugin.plugin_macro).

This should also be registered under `airflow.sdk.macros`:

```suggestion
        macros_module = make_module(f"airflow.sdk.macros.{plugin.name}", plugin.macros)
```

so add `macros` to `task-sdk/src/airflow/sdk/__init__.py`

Reply:
Makes sense. Let me add that.

Reply:
Handled in https://github.com/apache/airflow/pull/50642/commits/99ea71cbcb2d13762b424394007a1fea20ac5b21